### ISSUE TITLE
Repair transport

### ISSUE THREAD

**Issue by Kartaviy** [acemod/ACE3#2970]
You can make the possibility of repair parts for choice?

P.S
I am writing through an interpreter, sorry.

**Comment by PabstMirror** [comment]
I do not think I understood the question.  Could you rephrase it.

We do have a module for spare parts.
This module controls how many parts get added to a vehicle.
See: http://ace3mod.com/wiki/missionmaker/modules.html#2-1-adds-spare-parts

**Comment by headedhorseman101** [comment]
>You can make the possibility of repair parts for choice?

@Kartaviy Do you mean that you want the ability to repair only one part. As a example, the engine of a vehicle.

**Comment by Kartaviy** [comment]
"Do you mean that you want the ability to repair only one part? As an example, the engine of a vehicle."

Yes.Can you make invdividual menu for vehicles ,in which you can choose separate part for repairing. And also,pls make opportunity to prohibit repairing one or another part by choice of mission maker. One looming question - when you remove the caterpillar(track),why does wheel appear? Logically it should be corrected. Finally, please rename these words "Remove Wheel"  in these "Remove track".

**Comment by headedhorseman101** [comment]
>One looming question - when you remove the caterpillar(track),why does wheel appear?

Thank you for giving us this information. This can be made a bug report. Use this format to make bug reports: [http://ace3mod.com/wiki/user/how-to-report-an-issue.html] (http://ace3mod.com/wiki/user/how-to-report-an-issue.html)

>Can you make invdividual menu for vehicles ,in which you can choose separate part for repairing. And also,pls make opportunity to prohibit repairing one or another part by choice of mission maker.

Please make feature requests on #414.
